src/index.html

```diff
@@ -7,6 +7,131 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
+      <div
+        class="calendar__day calendar__day--start-day calendar__day--month-length--1"
+      >
+      </div>
```

Comment:
Should look like. And the main task is whenever I change modifier such as `start-day` or `month-length` the calendar should be changed.
If you need any help feel free to ask at fe_chat
```suggestion
    <div class="calendar calendar--start-day--mon calendar--month-length--30">
      <div class="calendar__day"></div>
```

Reply:
still not fixed